DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
@rjwang - the changes to this file need to be undone. the gangedME11a parameter is set for run2 via the postLS1Customs function (which will eventually be replaced by the eras.run2.... command here).